packages/core/src/layout/InfoCard/InfoCard.js

```diff
@@ -29,7 +29,13 @@ import BottomLink from './BottomLink';
 
 import textContent from 'react-addons-text-content';
 
-const BoldHeader = withStyles({ title: { fontWeight: '700' } })(CardHeader);
+const BoldHeader = withStyles({
+  title: { fontWeight: '700' },
+  subheader: { paddingTop: '2px' },
+  root: {
+    padding: '16px 16px 16px 20px',
```

Comment:
Same thing as above, please use `theme.spacing(2, 2, 2, 2.5)`

Reply:
oops, my mistake. You're right. Fixed. Thx! :smile: